.github/workflows/main.yml

```diff
@@ -29,107 +30,85 @@ jobs:
-          kind get kubeconfig --internal >$HOME/.kube/config
-          kubectl get nodes            
-
-      - name: Load image on the nodes of the cluster
-        run: |
-          kind load docker-image --name=kind litmuschaos/ansible-runner:ci
-
-      - name: Deploy a sample application for chaos injection
-        run: |
-          kubectl apply -f https://raw.githubusercontent.com/mayadata-io/chaos-ci-lib/master/app/nginx.yml
-          sleep 30
-          
-      - name: Setting up kubeconfig ENV for Github Chaos Action
-        run: echo ::set-env name=KUBE_CONFIG_DATA::$(base64 -w 0 ~/.kube/config)
-
-      - name: Running Litmus pod delete chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-pod-delete') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: pod-delete
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
+          working-directory: litmus-portal/frontend
 
-      - name: Running container kill chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-container-kill') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: container-kill
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          CONTAINER_KILL_HELPER_IMAGE: litmuschaos/container-kill-helper
-          CONTAINER_KILL_HELPER_TAG: latest
-          LIB: containerd
-          LITMUS_CLEANUP: true
+      - name: Starting the server
+        run: npm start &
+        working-directory: litmus-portal/frontend
 
-      - name: Running node-cpu-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-cpu-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-cpu-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
+      - name: Installing Cypress and running tests for login page
+        if: startsWith(github.event.comment.body, '/run-e2e-login')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/Login/*
+          working-directory: litmus-e2e/CypressE2E/
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
+
```

Comment:
Why did we comment it, is it required?

Reply:
Yes, it is required if we want the cypress dashboard to fetch the tests results and also for cypress bot to make comments.

Reply:
But first we have to install Cypress webhooks on litmus repo ,then only it will work that's why it is commented for now.

Reply:
Okay, I have removed them for now.